tests/cypress/support/commands.js

```diff
@@ -41,6 +41,40 @@ Cypress.Commands.add('userRegistration', (firstName, lastName, userName, emailAd
     }
 });
 
+Cypress.Commands.add('deletingRegisteredUsers', () => {
+    cy.request({
+        method: 'POST',
+        url: '/api/v1/auth/login',
+        body: {
+            username: Cypress.env('user'),
+            email: Cypress.env('email'),
+            password: Cypress.env('password'),
+        },
+    }).then(async (responce) => {
+        const authKey = await responce['body']['key'];
+        cy.request({
+            url: '/api/v1/users',
+            headers: {
+                Authorization: `Token ${authKey}`,
+            },
+        }).then(async (responce) => {
+            const responceResult = await responce['body']['results'];
```

Comment:
Probably we should use linear promise chain here (now you use nested promise handlers)):

```js
cy.request({
       ....
    }).then(() => {
       .....
       return cy.request({ ... })  
    }).then(() => {
       .......
       delete users in circle
    })
```

Reply:
Rework in a slightly different form.